src/core.js

```diff
@@ -1159,6 +1159,7 @@ define([
                 }
                 hidePageSpinner();
             } catch (e) {
+                window.console.log(util.formatExc(e));
```

Comment:
Yes, intended to keep it. Otherwise it's hard to debug errors in form loading logic.